src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -154,6 +154,7 @@ public abstract partial class JsonNamingPolicy
     {
         protected JsonNamingPolicy() { }
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
+        public static System.Text.Json.JsonNamingPolicy SnakeCase { get { throw null; } }
```

Comment:
Due to the many implementations of snake case, do we want to make it clear what algorithm we're using?

For example, instead of `JsonNamingPolicy.SnakeCase`, we could use `JsonNamingPolicy.SnakeCaseNewtonsoftCompat` or something similar to make that clear? I realize this passed API review, but we should revisit to ensure.

Reply:
@layomia I'd require an api design/review from you.